hw/top_darjeeling/top_darjeeling_jtag_id_pkg.core

```diff
@@ -5,6 +5,9 @@ CAPI=2:
 
 name: "lowrisc:constants:top_darjeeling_jtag_id_pkg"
 description: "jtag id for top_darjeeling"
+virtual:
+  - lowrisc:constants:jtag_id_pkg
```

Comment:
Same thing regarding a "virtual" hint in the name.

Reply:
Same as the virtual ibex_pmp_reset_pkg. It turns out there are no users of a virtual `lowrisc:constants:jtag_id_pkg`, so let's get rid of the virtual VLNV. :smile: